checks/evaluation/dependency_update_tool_test.go

```diff
@@ -88,14 +88,16 @@ func TestDependencyUpdateTool(t *testing.T) {
 					Tools: []checker.Tool{
 						{
 							Name: "DependencyUpdateTool",
-							File: &checker.File{
-								Path: "/etc/dependency-update-tool.conf",
-								Snippet: `
+							File: []checker.File{
+								{
+									Path: "/etc/dependency-update-tool.conf",
+									Snippet: `
 										[dependency-update-tool]
 										enabled = true
 										`,
-								Offset: 0,
-								Type:   0,
+									Offset: 0,
+									Type:   0,
```

Comment:
I'm not sure why Type is an integer, but it should probably be a `checker.SomeTypeName` instead.

Reply:
Yeah that's suspicious. I didn't modify the code in this part - only added a wrap since now File is an array. Let's discuss this tmr.